.editorconfig

```
@@ -22,4 +22,8 @@ dotnet_naming_symbols.private_fields.applicable_kinds           = field
 dotnet_naming_symbols.private_fields.applicable_accessibilities = private
 
 dotnet_naming_style.prefix_underscore.capitalization = camel_case
-dotnet_naming_style.prefix_underscore.required_prefix = _
\ No newline at end of file
+dotnet_naming_style.prefix_underscore.required_prefix = _
+[*.cs]
+
+# CS0659: Type overrides Object.Equals(object o) but does not override Object.GetHashCode()
+dotnet_diagnostic.CS0659.severity = silent

```

This is to suppress the error in `RelationshipAttribute.cs`, right? Are we sure it's fine to just ignore it rather than solving the issue? I don't know: I always ignored it until now, never looked into it.

---

I'll take it out, so it stays as a warning next release